apps/inviwo/CMakeLists.txt

```diff
@@ -47,8 +47,11 @@ target_link_libraries(inviwo PUBLIC
     inviwo::core
     inviwo::qtapplicationbase
     inviwo::qteditor
-    Qt5::Core
-    Qt5::Widgets
+    Qt${QT_VERSION_MAJOR}::Core
+    Qt${QT_VERSION_MAJOR}::Widgets
+)
+target_compile_definitions(inviwo PUBLIC 
+    "QT_DEPRECATED_WARNINGS_SINCE=QT_VERSION_CHECK(5, 15, 0)"
```

Comment:
Could add a comment explaining at which point this should be bumped or removed

Reply:
I don't know any more then when we move to require qt6. And that is quite obvious from the names...